src/gcloud_common_utils/blob_helper_local.py

```diff
@@ -18,21 +22,19 @@ def upload_blob(bucket_name: str, destination_blob_name: str, file_like_object):
     topic_name <bucket-name>-updates as a convention (emulating storage notifications)
     """
     logging.info(f"Writing file={destination_blob_name} to local filesystem")
-    temporary_bucket_name = "temp_file_upload"
-    file_path = _get_path(temporary_bucket_name, destination_blob_name)
-    directory, filename = os.path.split(file_path)
-    logging.info(f"Writing first to a temporary location {file_path}")
-    if not os.path.exists(directory):
-        os.makedirs(directory)
+    _, filename = os.path.split(destination_blob_name)
+
+    # Appending microsecond timestamp to filename for uniqueness in temp folder
+    file_path = _get_path(TEMP_BUCKET_NAME, filename + dt.datetime.now().strftime('_%H_%M_%S_%f'))
```

Comment:
filename is unique, isn't it? Boats as well, we made dirs just for clasififacion?

Reply:
filename's are not necessarily unique across buckets. Since we are using a single temp bucket for all bucket writes we should make the names unique just to be completely sure.

Reply:
Just checked that currently files are prefixed with the dir (FA. TF. etc....) But probably good to keep in case 
in the future we don't do that.